addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -279,6 +279,18 @@ weSnippetEditor.SnippetsMenu.include({
         return sel.rangeCount && [...this.getEditableArea()].some(el => el.contains(sel.anchorNode));
     },
 
+    /**
+     * The goal here is to disable parents editors for `s_popup` snippets
+     * since they should not display their parents options.
+     * TODO: Update in master to set the `o_no_parent_editor` class in the
+     * snippet's XML.
```

Comment:
We can leave this comment as it is for this stable fix but IMO in master, we should rather get rid of the `o_no_parent_editor` class entirely: I prefer less things added in the DOM and more things handled through registries.